cli/src/cli_util.rs

```diff
@@ -2796,11 +2796,11 @@ pub fn print_snapshot_stats(
             ui.hint_default(),
             r"
             This is to prevent large files from being added by accident. You can fix this by:
-              - Adding the file to `.gitignore`
-              - Run `jj config set --repo snapshot.max-new-file-size {size}`
-                This will increase the maximum file size allowed for new files, in this repository only.
-              - Run `jj --config snapshot.max-new-file-size={size} st`
-                This will increase the maximum file size allowed for new files, for this command only.
+            - Adding the file to `.gitignore`
+            - Run `jj config set --repo snapshot.max-new-file-size {size}`
+              This will increase the maximum file size allowed for new files, in this repository only.
+            - Run `jj --config snapshot.max-new-file-size={size} st`
+              This will increase the maximum file size allowed for new files, for this command only.
```

Comment:
The extra indentation for the list here is intentional, isn’t it?

Reply:
Yeah, I guess that indentation was added to achieve the same thing that now happens for all log messages anyway.

before:
```
Hint: This is to prevent large files
  - Adding the file to `.gitignore`
  - Run `jj config set
```

after, with manual indentation kept:
```
Hint:
  This is to prevent large files
    - Adding the file to `.gitignore`
    - Run `jj config set
```

after, with manual indentation removed:
```
Hint:
  This is to prevent large files
  - Adding the file to `.gitignore`
  - Run `jj config set
```

The latter seems nicer to me but I'll revert if the former is preferred.

Reply:
I guess I'm neutral. In YAML I find unindented lists annoying, but in Markdown they're fine.